code/modules/shuttle/emergency.dm

```diff
@@ -86,13 +86,13 @@
 
 	if((old_len != authorized.len) && !ENGINES_STARTED)
 		var/alert = (authorized.len > old_len)
+		var/repeal = (authorized.len < old_len)
 		var/remaining = auth_need - authorized.len
 		if(authorized.len && remaining)
 			minor_announce("[remaining] authorizations \
 				needed until shuttle is launched early", null, alert)
-		else
-			minor_announce("All authorizations to launch the shuttle \
-				early have been revoked.")
+		if(repeal)
+			minor_announce("Early launch authorization revoked [remaining] authorizations needed")
```

Comment:
Early launch authorization revoked, [remaining] authorizations needed

Put a comma in.